client/extensions/woocommerce/app/dashboard/store-move-notice-view.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+
+import { Card, Button } from '@automattic/components';
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
+import config from 'calypso/config';
+
+/**
+ * Image dependencies
+ */
+
+import megaphoneImage from 'calypso/assets/images/woocommerce/megaphone.svg';
+
+class StoreMoveNoticeView extends Component {
+	render = () => {
+		const { site, title, status } = this.props;
+		return (
+			<Card className={ classNames( 'dashboard__store-move-notice', status ) }>
+				<img src={ megaphoneImage } alt="" />
+				<h1>{ title }</h1>
+				<p>
+					Now you'll be able to access all of your most important store management features in one
+					place.{ ' ' }
```

Comment:
Using `{ ' ' }` feels odd, but since this will go away once we add the `translate` wrapping back in, I think it is fine.

Reply:
@mattsherman I actually did not even know I could do that :) It was added by jslint when I committed. Thank you for the review! I will merge it next week.